### ISSUE TITLE
`npm run all` fails on Node 17.1

### ISSUE THREAD

**Issue by emersonveenstra** [facebookincubator/RapiD#397]
`npm run all` on Node 17.1 results in this error:
```
> RapiD@1.1.8 build:data
> shx mkdir -p dist/data && node scripts/build_data.js

node:internal/errors:464
    ErrorCaptureStackTrace(err);
    ^

TypeError [ERR_IMPORT_ASSERTION_TYPE_MISSING]: Module "file:///home/emerson/repos/test/RapiD/node_modules/cldr-core/supplemental/territoryInfo.json" needs an import assertion of type "json"
    at new NodeError (node:internal/errors:371:5)
    at validateAssertions (node:internal/modules/esm/assert:74:15)
    at defaultLoad (node:internal/modules/esm/load:25:3)
    at ESMLoader.load (node:internal/modules/esm/loader:353:26)
    at ESMLoader.moduleProvider (node:internal/modules/esm/loader:274:58)
    at new ModuleJob (node:internal/modules/esm/module_job:66:26)
    at ESMLoader.#createModuleJob (node:internal/modules/esm/loader:291:17)
    at ESMLoader.getModuleJob (node:internal/modules/esm/loader:255:34)
    at async ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:81:21)
    at async Promise.all (index 11) {
  code: 'ERR_IMPORT_ASSERTION_TYPE_MISSING'
}

Node.js v17.1.0
ERROR: "build:data" exited with 1.
ERROR: "build" exited with 1.
```

which is apparently [a new error added in Node 17.1](https://nodejs.org/api/errors.html#err_import_assertion_type_missing). 

I opened this here instead of in iD because the latest commit of iD at this point (f91ae8902fd17f8e9477c6d4c11cbf8ff96158a5) builds fine.

**Comment by Bonkles** [comment]
I was able to build RapiD just fine from the `main` branch just now using node v17.1. 

```Now using node v17.1.0 (npm v6.14.13)
clarkben@clarkben-mbp ~ % cd working/RapiD 
clarkben@clarkben-mbp RapiD % npm run clean; npm install; npm run all```

I got a message about the lockfile being slightly incompatible with the new version of node, but the build did complete. Admittedly, we haven't looked much at npm 17 yet. 

If it helps, the build I ran seems to have pulled in the following version of cldr-core: 
```

    "cldr-core": {
      "version": "39.0.0",
      "resolved": "https://registry.npmjs.org/cldr-core/-/cldr-core-39.0.0.tgz",
      "integrity": "sha512-ZSElD4Ckl2W3ApP+jz8XXOWen+UqELaO76P0J1H6SKxWuwkba1zmE5gU0grNyuI1RAYwBRUhbeIV+YTsJ7yw3g==",
      "dev": true
    },

```

**Comment by bhousel** [comment]
Oh yeah this was related to the new import assertions syntax for importing .json files in node.
This ended up getting fixed around 0a1628877
